config/database.yml

```diff
@@ -11,15 +11,15 @@ default: &default
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: /db/development.sqlite3
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: /db/test.sqlite3
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: /db/production.sqlite3
```

Comment:
These URIs were adapted to use docker volumes. 
pls inform if any further action is needed here